src/biconnectivity/biconnect.jl

```diff
@@ -0,0 +1,73 @@
+"""
+Biconnections: A state type for Depth First Search that finds the biconnected components
+"""
+type Biconnections
+    low::Vector{Int}
+    depth::Vector{Int}
+    stack::Vector{Edge}
+    biconnected_comps::Vector{Vector{Edge}}
+    id::Int
+end
+
+function Biconnections(g::SimpleGraph)
+    n = nv(g)
+    return Biconnections(zeros(Int, n), zeros(Int, n), Vector{Edge}(), Vector{Vector{Edge}}(), 0)
+end
+
+"""
+Computes the biconnected components of a graph `g`
+and returns a Vector of vectors containing each biconnected component
+"""
+function biconnected_components(g::SimpleGraph)
```

Comment:
I don't think this Tarjan algorithm implementation could be used for directed graphs.